quickstart/src/miclevel.js

```diff
@@ -43,9 +49,9 @@ module.exports = audioContext ? function micLevel(stream, maxLevel, onLevel) {
         onLevel(level);
       }
 
-      requestAnimationFrame(track.readyState === 'ended'
-        ? () => onLevel(0)
-        : checkLevel);
+      requestAnimationFrame(audioTrack.readyState === 'ended'
```

Comment:
`audioTrack.readyState` doesn't exist, so this logic isn't doing anything. Looks like it should be `audioTrack.mediaStreamTrack.readyState === 'ended'` instead.

However, if we stop the animation loop when the `readyState === 'ended'`, I think we will still see the same problem as before. I think we have two approaches here:

1. Use `requestAnimationFrame(checkLevel)`. This way the animation loop doesn't stop when a track stops.
2. Update `initializeAnalyser` so that it restarts the animation loop whenever the track is started. 

 I'm not really sure what would be best here. Option 1 is simplest, but I'm not sure if it would cause problems. What do you think @PikaJoyce?

Reply:
Thanks for the suggestion @timmydoza!

Yes, I have actually tried both of those options. Not concurrently, but it doesn't really seem like either did any damage. I can try both and see which one is most impactful and get back to you which one I think would work better.